ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -31,6 +35,24 @@ sub Configure($$@) {
     # Define paths for convenience.
     my $base = "/software/components/dirperm";
 
+    my $filesystems = "/system/filesystems";
+    if ($config->elementExists($filesystems)) {
```

Comment:
hmmm, ugly `Element` code and `$_` usage...
```perl
my $fs_tree = $config->getTree($filesystems);
push(@configured_mounts, map {$_->{mountpoint}} 
                                               sort { $a->{mountpoint} cmp $b->{mountpoint}}
                                               @{$fs_tree});
```
requires some recent CCM (15.8 i think) and is completely untested :wink: